trivial/flake.nix

```diff
@@ -1,6 +1,10 @@
 {
   description = "A very basic flake";
 
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs?ref=nixos-unstable";
```

Comment:
```suggestion
    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
```

Reply:
the `/` doesn't work if your git branch contains a `/`  while `?ref=` universally works. might be best to showcase the sure method instead of the "might work" one ?

Reply:
Just wanted to note that branches containing slashes has worked for me for years